content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+Pipeline: API,
+Pipeline: Basic Steps,
+Pipeline: Durable Task Step, 
+Pipeline: Execution Support,
+Pipeline: Global Shared Library for CPS pipeline,
+Pipeline: Groovy CPS Execution,
+Pipeline: Job,
+Pipeline: SCM Step,  
+Pipeline: Step API
+| required
 
-## Running your Pipeline
+| Pipeline: Stage View
+| Provides a graphical swimlane view of pipeline stage execution (as well as a build history of the stages)
+| recommended
 
-Because pipelines are built as Jenkins jobs, they can be built like other jobs. You can use the Build Now feature to manually trigger your build on demand or set up triggers to execute your pipeline based on certain events.
+| Multibranch Pipeline
+| Adds "Multibranch Pipeline" item type which allows Jenkins to automatically build branches that contain jenkinsfile
+| recommended
 
-## Adding Stages and Steps
-Stages are usually the top most element of Pipeline syntax. Stages allow you to group your build step into its component parts. By default, multiple builds of the same pipeline can run concurrently. The stage element also allows you to control this concurrency:
+| CloudBees GitHub Branch Source
+| Adds GitHub Organization Folder item type and adds "Github" as a branch source on Multibranch pipelines
+| recommended for teams hosting repositories in Github
 
-```groovy
-stage 'build'
-   node{ … }
-stage name: 'test', concurrency: 3
-   node{ … }
-stage name: 'deploy', concurrency: 1
-   node{ … }
-```
+| CloudBees Bitbucket Branch Source
+| Adds Bitbucket Team item type and adds "Bitbucket" as a branch source on Multibranch pipelines
+| recommended for teams hosting repositories in Bitbucket
 
-In this example, we have set a limit of three concurrent executions of the test stage and only one execution of the deploy stage. You will likely want to control concurrency to prevent collisions (for example, deployments).
+| CloudBees Docker Pipeline
+| Enables pipeline to build and use Docker containers inside pipeline scripts.
+| optional
 
-Newer builds are always given priority when entering a throttled stage; older builds will simply exit early if they are preempted.
+|=======================
 
-### General Build Steps
 
-Within your stages, you will add build steps. Just like with Freestyle Jenkins jobs, build steps make up the core logic of your pipeline. Jenkins Pipeline supports any compatible Build Step and populates the snippet generator with all available Build Steps in your Jenkins environment.
 
-```groovy
-step([$class: 'JavadocArchiver', javadocDir: 'target/resources/javadoc', keepAll: false])
-```
+Pipeline-related plugins other than those listed above are regularly “whitelisted” as compatible with or designed for Pipeline usage. For more information, see the link:https://github.com/jenkinsci/workflow-plugin/blob/541faf611659e1e6b8f2cbbd3435756b27633db4/COMPATIBILITY.md[Plugin Compatibility With Pipeline] wiki. Current contents of that wiki are summarized in the following table:
```

Comment:
I would rather us not include this list here at all, it's guaranteed to get out of date and I don't have the time to constantly keep it updated (I would assume neither do the Pipeline developers).

It would be better, in my opinion, to just link off to the document in the Pipeline Git repository.